# ImageArray.msg
sensor_msgs/Image[] images